fastlane/docs/Guide.md

```diff
@@ -40,15 +38,15 @@ Install the gem and all its dependencies (this might take a few minutes).
 
 ## Setting up `fastlane`
 
-Before changing anything, I recommend committing everything in `git` (in case something goes wrong).
+Before changing anything, I recommend committing everything in `git`.
```

Comment:
Before making any modifications to your project, it's always recommended to commit your current directory in `git`.